### PR TITLE
Release capa v2.0 🎉 

### PR DESCRIPTION
**Pull request by Ana06**
# Release checklist

- [X] Ensure all [milestoned issues/PRs](https://github.com/fireeye/capa/milestones) are addressed, or reassign to a new milestone.
- [X] Add the `dont merge` label to all PRs that are close to be ready to merge (or merge them if they are ready) in [capa](https://github.com/fireeye/capa/pulls) and [capa-rules](https://github.com/fireeye/capa-rules/pulls).
- [X] Ensure the [CI workflow succeeds in master](https://github.com/fireeye/capa/actions/workflows/tests.yml?query=branch%3Amaster).
- [x] Ensure that `python scripts/lint.py rules/ --thorough` succeeds (only `missing examples`  offenses are allowed in the nursery). 
- [X] Review changes
  - capa https://github.com/fireeye/capa/compare/\<last-release\>...master
  - capa-rules https://github.com/fireeye/capa-rules/compare/\<last-release>\...master
- [X] Update [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md)
  - Do not forget to add a nice introduction thanking contributors
  - Remember that we need a major release if we introduce breaking changes
  - Sections: see template below
  - Update `Raw diffs` links
  - Create placeholder for `master (unreleased)` section
    ```
    ## master (unreleased)

    ### New Features

    ### Breaking Changes

    ### New Rules (0)

    -

    ### Bug Fixes

    ### capa explorer IDA Pro plugin

    ### Development

    ### Raw diffs
    - [capa <release>...master](https://github.com/fireeye/capa/compare/<release>...master)
    - [capa-rules <release>...master](https://github.com/fireeye/capa-rules/compare/<release>...master)
    ```
- [X] Update [capa/version.py](https://github.com/fireeye/capa/blob/master/capa/version.py)
- [x] Create a PR with the updated [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md) and [capa/version.py](https://github.com/fireeye/capa/blob/master/capa/version.py). Copy this checklist in the PR description.
- [ ] After PR review, merge the PR and [create the release in GH](https://github.com/fireeye/capa/releases/new) using text from the [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md).
- [ ] Verify GH actions [upload artifacts](https://github.com/fireeye/capa/releases), [publish to PyPI](https://pypi.org/project/flare-capa) and [create a tag in capa rules](https://github.com/fireeye/capa-rules/tags) upon completion.
- [ ] [Spread the word](https://twitter.com) **We should wait for the blog post for this**
- [ ] Update internal service

### PR Checklist

- [ ] No CHANGELOG update needed

- [X] No new tests needed

- [ ] No documentation update needed

### CODE REVIEW

File: capa/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "1.6.1"
+__version__ = "2.0.0"
```
Comment (mr-tz) [inline capa/version.py:1]: 🥳

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,32 @@
...
+- [capa v2.0.0...master](https://github.com/fireeye/capa/compare/v2.0.0...master)
+- [capa-rules v2.0.0...master](https://github.com/fireeye/capa-rules/compare/v2.0.0...master)
+
+
+## v2.0.0 (2021-06-30)
+
+We are excited to announce version 2.0! :tada:
+capa 2.0 enables anyone to contribute rules more easily and it is the first Python 3 ONLY version.
+It includes many features and enhancements for the capa explorer IDA plugin and more concise and relevant result via identification of library functions using FLIRT.
+It also adds 89 new rules, including all new techniques introduced in MITRE ATT&CK v9.
```
Comment (mr-tz) [inline CHANGELOG.md:29]:
```suggestion
capa 2.0:
- enables anyone to contribute rules more easily
- is the first Python 3 ONLY version
- includes many features and enhancements for the capa explorer IDA plugin
- provides more concise and relevant result via identification of library functions using FLIRT
- adds 89 new rules, including all new techniques introduced in MITRE ATT&CK v9
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,38 @@
...
+
+We are excited to announce version 2.0! :tada:
+capa 2.0:
+- enables anyone to contribute rules more easily
+- is the first Python 3 ONLY version
+- includes many features and enhancements for the capa explorer IDA plugin
+- provides more concise and relevant result via identification of library functions using FLIRT
+- adds 89 new rules, including all new techniques introduced in MITRE ATT&CK v9
+
+We appreciate everyone who opened issues, provided feedback, and contributed code and rules.
```
Comment (mr-tz) [inline CHANGELOG.md:35]:
```suggestion
A huge thanks to everyone who submitted issues, provided feedback, and contributed code and rules. Many colleagues across dozens of organizations have volunteered their experience to improve this tool!
```

**Comment by ankhazam** [comment]
Hi, what's with the delay, when can we expect the great merge? ;)

**Comment by mr-tz** [comment]
We're almost ready and very excited to finally hit that green button 😉

**Comment by williballenthin** [comment]
fireeye/siglib is public: https://github.com/fireeye/siglib

**Comment by ankhazam** [comment]
Hey, I tried building the 2.0.0 code locally as well as installing capa from pypi into a docker container. In both scenarios I ended up with the following error after running capa:
me@my-pc: capa -r /capa-rules /files/sample.exe
loading : 100%|██████████| 579/579 [00:00<00:00, 1645.93 rules/s]
ERROR:capa:signatures path /usr/local/lib/python3.9/site-packages/capa/../sigs does not exist or cannot be accessed

**Pull request merged**